### ISSUE TITLE
Mapped task names are not grouped on dask / bokeh

### ISSUE THREAD

**Issue by JacksonMaxfield** [PrefectHQ/prefect#2821]
## Description
*A clear description of the bug*
See #2813 for original context.

From the 0.12.0 release task names weren't being inferred / passed properly to the dask scheduler. That was fixed for single tasks after #2814 was merged. However after just testing out current git head it seems like mapped tasks are copied over _with_ their index in the name which results in `bokeh` looking like:

```
mapped_task [0] 1/1
mapped_task [1] 1/1
mapped_task[2] 1/1
...
```

## Expected Behavior
*What did you expect to happen instead?*

Previous behavior in 0.11.0 where mapped tasks were all named the same thing and so `bokeh` had a single progress bar for them all.

## Reproduction
*A minimal example that exhibits the behavior.*

```python
from distributed import Client
from prefect import Flow, task
from prefect.engine.executors import DaskExecutor

@task
def add_one(x):
    return x + 1

with Flow("test") as flow:
    values = add_one.map(range(10))

cluster = LocalCluster()
client = Client(cluster)

print(f"Dashboard available at: {client.dashboard_link}")

flow.run(executor=DaskExecutor(address=cluster.scheduler_address))
```

## Environment
*Any additional information about your environment*

`prefect`: current git head
`dask[bag]==2.18.1`
`distributed==2.18.0`
`bokeh==2.0.2`

*Optionally run `prefect diagnostics` from the command line and paste the information here*

**Comment by jcrist** [comment]
Thanks for the issue report, should be fixed by #2831.

**Comment by JacksonMaxfield** [comment]
Thanks a lot!
